integration/only-types/parameters.txt

```diff
@@ -0,0 +1 @@
+outputEncodeMethods=false,outputJsonMethods=false,outputClientImpl=false
```

Comment:
Huh, do you know why these `integration/only-types` files are showing up as new in this PR? They're already on master...

Reply:
Maybe I hadn't pushed out the `master` with your previously-merged PR. Maybe make sure you're rebased on top of the latest `master`?